exercise/map.js

```diff
@@ -0,0 +1,56 @@
+/*
+## Map
+
+### Instructions
+
+Create an array named 'arr2' that contains the values of 'arr1'
+with the following modifications applied to each of the values:
+1) They have to be multiplied by 2
+2) Then the results have to be subtracted by 4
+3) finally, the negative values have to be replaced by null
+then do the same with arrRandom1 by returning arrRandom2
+The use of `for` and `while` is forbidden for this exercise
+
+### Notions
+
+- Map
+
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
+
+*/
+let arr1 = [10, -10, 20, -95, 86]
+///*/ // ⚡
+
+///*/// ⚡
+
+let arrRandom1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+for (let i = 0; i < 10; i++) {
+  let value = Math.round(Math.random() * (10 - -10 - 10))
+  arrRandom1.push(value)
```

Comment:
If you need random values in your tests, it's best not to expose them outside of tests.
For that you can use test context:

```js
t((_, ctx) => ctx.arrRandom1 = Array(10).fill().map(() => Math.round(Math.random() * 10)))
```
you can then use your context like so:
```js
t(({ eq }, ctx) => eq(fn(ctx.arrRandom1)[0], check(arr[0]))
```

Reply:
Noted. Declaring them in the setup now.